### PR TITLE
Contribution

### LINKED ISSUE #125: I want to add a calculator with more than just basic functions using C language
Please include a small description or summary of the issue here.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

**Languages**
- [ ] Python
- [ ] Java
- [x] C
- [ ] C++

I will be adding a calculator with sqrt, log, power func along with basic operations using C language
[c calculator for github.txt](https://github.com/kaustubhgupta/Hacktoberfest2020_/files/5319449/c.calculator.for.github.txt)

### PR DESCRIPTION
**Pull request by adilsp5075**
# my_calculator

I create a calculator with c programming which contains six functions including the basic four functions
it is a menu driven program 

Fixes #125 

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by adilsp5075** [comment]
I hope that I will follow the instructions correctly this time

File: C/README.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ C is a general-purpose, high-level language that was originally developed by Den
 |  Sum of digits of number   |                  [link](./sumOfNumberDigits.C)                   |
 |  Binary Search Tree        |                  [link](./BinarySearchTree.C)                    |
 | Floyd's Cycle Detection    |          [link](./Floyds_cycle_detection_for_linked_lists.c)     |
-|  Hash Table Implementation |                  [link](./HashTableImplementation.c)             | 
\ No newline at end of file
+|  Hash Table Implementation |                  [link](./HashTableImplementation.c)             | 
+| calculator with extra functions|              [link](./my_calculator.c)                       |
```
Comment (kaustubhgupta) [inline C/README.md]: The link here is incorrect. It should be (./my calc/my_calculator.c)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@adilsp5075 Fill the PR template

**Comment by adilsp5075** [comment]
Sorry for all these things
I will make it right

**Comment by adilsp5075** [comment]
I resolve the problem did you get it?

**Review by kaustubhgupta — APPROVED** [review]

**Comment by adilsp5075** [comment]
thank you :)

**Comment by kaustubhgupta** [comment]
@adilsp5075 Thanks for contributing!

**Pull request merged**